gapis/api/vulkan/transform_mapping_exporter.go

```diff
@@ -105,9 +109,55 @@ func (mappingTransform *mappingExporter) TransformCommand(ctx context.Context, i
 		mappingTransform.thread = inputCommands[0].Thread()
 	}
 
+	// This is a workaround for b/180532468.
```

Comment:
I think it would be easier for the reader to get an idea what's going on here to rephrase this. Particularly, give a short bit of context what's going on and introduce things in order.

For example, from what I think is going on here, this may be easier to read:

// Vulkan allows duplicate handles which we don't track properly (b/180532468).
// As a partial workaround, keep track of the used handles outside of `initialCmds`
// to reduce the chance of mapping to the wrong object.